### ISSUE TITLE
[BUG] - [D&D5e] - Sharing settings not saved when dragging item pile actor to scene

### ISSUE THREAD

**Issue by missmelisande** [fantasycalendar/FoundryVTT-ItemPiles#464]
**Have you tried to reset Item Piles' system settings?**
Yes

**Describe the bug**
I have modified the sharing settings on an Item Pile that I want to be able to reuse to make multiple piles, but when I drag the actor to the scene, the sharing settings reset to default and have to be manually changed for each item pile.

**To Reproduce**

1. Create an item pile actor.
2. In the token settings, make sure Link Actor Data is unchecked.
3. Modify the Sharing Settings to anything other than the default settings.
4. Drag the actor to the scene and open the Sharing Settings - they will have reverted to the default settings.

**Expected behavior**
I would expect the sharing settings to be maintained so they don't have to be manually changed for every Item Pile.

**Setup:**

- Foundry Version: 11.308
- System Version: DND5e 2.3.1
- Item Piles Version: 2.7.14
- Browser & version: Google Chrome 117.0.5938.92

**Active modules:**

- Item Piles
- libwrapper
- socketlib

**Comment by Haxxer** [comment]
Can you confirm you have absolutely no other modules active than the ones you listed?

**Comment by missmelisande** [comment]
Yeah, I tried it on a test world that's basically empty and only has Item Piles and its necessary modules active.

**Comment by Haxxer** [comment]
I cannot reproduce this - on a fresh DND5e world, I can create a new NPC character with a few item pile settings configured:

![image](https://github.com/fantasycalendar/FoundryVTT-ItemPiles/assets/12858387/0ce549ad-d278-435b-a63c-b0b8bf75a26f)

When I drag this onto the scene, and check its item pile settings, it is identical:

![image](https://github.com/fantasycalendar/FoundryVTT-ItemPiles/assets/12858387/7052c426-f3ee-42ae-84c6-1348162de32f)

**Comment by missmelisande** [comment]
So, I tested things out a bit more. I updated Foundry to 11.311. The default item pile that got automatically created still has the behaviour where it loses the sharing settings in both my actual game and the test game that I set up with no modules (maybe that's intentional?) when I drag from the actors tab onto a scene. I also have a second item pile in my actual game that I made previously that does this also. 
However, I just went in to my actual game and made a completely new Item Pile from scratch and it seems to retain the sharing settings when dragged out to a scene.
